src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {  BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
-import HomePage from './page/home/HomePage';
+import Home from './pages/Home/Home';
```

Comment:
```suggestion
import Home from './pages/Home';
```
hai cái dưới cũng tương tự em nhé